script.js

```diff
@@ -1,4 +1,262 @@
+    return `You got the ${playerHand[0].name} of ${playerHand[0].suit} and the ${playerHand[1].name} of ${playerHand[1].suit}. The dealer got the ${dealerHand[0].name} of ${dealerHand[0].suit} and another card.<br>Please input 'hit' or 'stand' as your choice.`;
+  }
+
+  if (getSumOfHand(playerHand) == maxSumOfCards) {
+    endGame = true;
+    return "Player wins game by having a total of 21.";
+  }
+  if (getSumOfHand(dealerHand) == maxSumOfCards) {
+    endGame = true;
+    return "Dealer wins game by having a total of 21.";
+  }
+
+  // if (playerChoseStand == true) {
+  //   playerChoseStand = true;
+  //   if (input != "hit" && input != "stand") {
+  //     return "Please input either 'hit' or 'stand' to continue.";
+  //   }
+
+  if (input == "hit") {
+    playerHand = dealCardToHand(playerHand);
+    index2 += 1;
+    if (getSumOfHand(playerHand) > maxSumOfCards) {
+      return `You got the ${playerHand[0].name} of ${playerHand[0].suit} and the ${playerHand[1].name} of ${playerHand[1].suit} ${others}. The dealer got the ${dealerHand[0].name} of ${dealerHand[0].suit} and another card.<br> Dealer wins! You have a sum of more than 21 and lost. Please try again.`;
+    } else {
+      others += `and the ${playerHand[index2].name} of ${playerHand[index2].suit}`;
+      return `You got the ${playerHand[0].name} of ${playerHand[0].suit} and the ${playerHand[1].name} of ${playerHand[1].suit} ${others}. The dealer got the ${dealerHand[0].name} of ${dealerHand[0].suit} and another card<br>Please input 'hit' or 'stand' as your choice.`;
+    }
+  }
+
+  if (input == "stand") {
+    playerChoseStand = true;
+  }
+  // }
+
+  if (playerHand[0].rank == "1" || playerHand[1].rank == "1") {
+    // `Please input '1' or '11' for your ace card.`;
+    if (input == "1") {
+      aceValue = 1;
+      return `The value of ace card is now set to '1'.`;
+    } else if (input == "11") {
+      aceValue = 11;
+      return `The value of ace card is now set to '11'.`;
+    }
+    //ASK THIS ONE
+    //return `Please input '1' or '11' into the box.`
+  }
+
+  // if (playerChoseStand) {
+  //   if (acesInHand > 0) {
+  //     ace = true;
+  //   }
+  // }
+
+  // if (ace && (input != "11") & (input != "1")) {
+  //   return `Do you want your value of ace to be '1' or '11'? `;
+  // }
+  // if ((ace && input == "1") || (ace && input == "11")) {
+  //   if (input == "11") {
+  //     difference = 11;
+  //     // acesInHand = 0;
+  //     ace = false;
+  //     return `The value of ace has been set to 11`;
+  //   } else {
+  //     // acesInHand = 0;
+  //     ace = false;
+  //     return `The value of ace has beens set 1`;
+  //   }
+  // }
+
+  console.log(getSumOfHand(playerHand));
+  console.log(getSumOfHand(dealerHand));
+
+  if (ace == false) {
+    if (getSumOfHand(dealerHand) < 17) {
+      dealerHand = dealCardToHand(dealerHand);
+      console.log(getSumOfHand(playerHand));
+      console.log(getSumOfHand(dealerHand));
+    }
```

Comment:
you might want to use a while loop instead because dealing one more card might not make the dealer hand greater than 17